core/gui/src/app/workspace/component/workflow-editor/context-menu/context-menu/context-menu.component.ts

```diff
@@ -50,4 +54,18 @@ export class ContextMenuComponent {
       .pipe(untilDestroyed(this))
       .subscribe(modifiable => (this.isWorkflowModifiable = modifiable));
   }
+
+  public checkIfOperatorHasAnyResult(): boolean {
+    const highlightedOperatorIDs = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedOperatorIDs();
+    const resultService = this.workflowResultService.getResultService(highlightedOperatorIDs[0]);
+    return (
+      resultService?.getCurrentResultSnapshot() !== undefined ||
+      this.workflowResultService.hasAnyResult(highlightedOperatorIDs[0])
```

Comment:
Multiple operators can be highlighted (using shift + click) but your logic only handles a single operator. You should fix this.

Reply:
Multiple operator download option added